packages/sitecore-jss-react/src/components/Image.tsx

```diff
@@ -139,11 +139,14 @@ export const getEEMarkup = (
   // We do not attempt to merge.
   const imgAttrs = getImageAttrs({ ...foundImgProps, ...otherProps }, imageParams, mediaUrlPrefix);
 
+  // console.log(imgAttrs);
   if (!imgAttrs) {
     return getEditableWrapper(imageField.editable);
   }
 
-  const imgHtml = ReactDOMServer.renderToStaticMarkup(<img {...imgAttrs} />);
+  const imgHtml = `<img ${getAttributesString(
+    imgAttrs as { [key: string]: string | number | boolean | Record<string, unknown> }
```

Comment:
To simplify this cast action, the getAttributesString can accept _{ [key: string]: unknown }_. In this case, you don't need to cast the type, and I'm not sure that it will make _getAttributesString_ more complex, it also can accept the same type

Reply:
Hmm yes you're right. This simplified it.